### ISSUE TITLE
Improve README Accessibility

### ISSUE THREAD

**Issue by ajloria** [sendgrid/sendgrid-csharp#825]
The README table of contents is missing some links. This needs to be added for better accessibility and ease.

**Comment by ajloria** [comment]
I can fix this issue.

**Comment by ajloria** [comment]
@thinkingserious or any Sendgrid admins, can this be part of the hacktoberfest label please? Thanks.
